### ISSUE TITLE
Plans to add UINMF?

### ISSUE THREAD

**Issue by LArnoldt** [welch-lab/pyliger#18]
Thank you for this great package! Are there any plans to port UINMF as a python version?

**Comment by theAeon** [comment]
Yeah, I'm working on a unified NMF library over [here](https://github.com/welch-lab/libplanc) and UINMF will absolutely be part of it!

**Comment by theAeon** [comment]
Just as a quick update on this: https://github.com/welch-lab/pyliger/tree/pyplanc-poc is where the bones of a UINMF integration are being put into place.

Unfortunately, my grant ran out, so I'm not going to be on this during work hours for the foreseeable future.
